src/components/A11yNotifications.vue

```diff
@@ -0,0 +1,47 @@
+//this component is to show the notifications
+<template>
+    <div class="screenreader-only">
+        <div aria-live="assertive">
+            {{ assertiveNotification }}
+        </div>
+        <div aria-live="polite">
+            {{ politeNotification }}
+        </div>
+    </div>
+</template>
+<script>
+import { store } from "@/store";
+export default {
+    name: "A11yNotifications",
+    computed: {
+        assertiveNotification() {
+            return store.getAccessibilityNotification("assertive");
+        },
+        politeNotification() {
+            return store.getAccessibilityNotification("polite");
+        },
+    },
+};
+</script>
+<style scoped>
+.screenreader-only {
+    position: absolute;
```

Comment:
You shouldn't need to define this, there's a .screenreader-only class at the top level in a global scope, would be best to use that.